DepositeCalcTests/Tests/CalculatorPageTests.cs

```diff
@@ -0,0 +1,141 @@
+        {
+            // Arrange
+            var calculatorPage = new CalculatorPage(driver);
+
+            // Act
+            calculatorPage.FinancialYear = financialYear;
+            calculatorPage.FillingMandatoryTextFields(depositAmount, interestRate, investmentTerm);
+            calculatorPage.Calculate();
+
+            // Asserts
+            Assert.AreEqual(expectedIncome, calculatorPage.Income, "Incorrect value in the Income field");
+            Assert.AreEqual(expectedInterestEarned, calculatorPage.InterestEarned, "Incorrect value in the Interest earned field");
+        }
+
+        [TestCase("360", "1000001", "10", "360")]
+        [TestCase("365", "1000000", "101", "365")]
+        [TestCase("360", "1000000", "100", "361")]
+        [TestCase("365", "1000000", "100", "366")]
+        public void NegativeCalculationTest(string financialYear, string depositAmount, string interestRate, string investmentTerm)
+        {
+            // Arrange
+            var calculatorPage = new CalculatorPage(driver);
+
+            // Act
+            calculatorPage.FinancialYear = financialYear;
+            calculatorPage.FillingMandatoryTextFields(depositAmount, interestRate, investmentTerm);
+
+            // Assert
+            Assert.IsTrue(calculatorPage.IsCalculateBtnDisabled, "Calculate button is clickable. This means that an invalid value may be entered in one of the text fields");
+        }
+
+        [TestCase(2024, 2)]
+        [TestCase(2028, 2)]
+        [TestCase(2025, 9)]
+        [TestCase(2029, 12)]
+        public void DayFieldTests(int year, int month)
+        {
+            // Arrange
+            var calculatorPage = new CalculatorPage(driver);
+            var expectedDays = DateHelper.GetMonthDays(year, month);
+
+            // Atc
+            calculatorPage.StartDateYear = year.ToString();
+            calculatorPage.StartDateMonth = DateHelper.NumberOfMonthToNameOfMonth(month.ToString());
+
+            // Assert
+            Assert.AreEqual(expectedDays, calculatorPage.GetStartDateDayList(), "Incorrect value in the day field");
+        }
+
+        [Test]
+        public void MonthFieldTests()
+        {
+            // Arrange
+            var calculatorPage = new CalculatorPage(driver);
+            var expectedMonthes = new[]
+            {
+                "January",
+                "February",
+                "March",
+                "April",
+                "May",
+                "June",
+                "July",
+                "August",
+                "September",
+                "October",
+                "November",
+                "December"
+            };
+
+            // Assert
+            Assert.AreEqual(expectedMonthes, calculatorPage.GetStartDateMonthList(), "Incorrect value in the month field");
+        }
+
+        [TestCase(2010, 2029)]
+        public void YearsFieldTests(int minimalYear, int maximumYear)
```

Comment:
You don't need to specify parameters, if there is only one test.
```suggestion
        [Test]
        public void YearsFieldTests()
```